cmd/nodecmd/create_gcp.go

```diff
@@ -100,37 +100,38 @@ func getGCPCloudCredentials() (*compute.Service, string, string, error) {
 	return computeService, gcpProjectName, gcpCredentialsPath, err
 }
 
-func getGCPConfig(zone string) (*compute.Service, string, string, string, string, error) {
-	if zone == "" {
+func getGCPConfig(zones []string) (*compute.Service, []string, string, string, string, error) {
+	if len(zones) == 0 {
```

Comment:
same here as for aws. maybe we can have just one function to get regions + num nodes

Reply:
agree